sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.0.0 (2020-10-30)
+
+- The is the GA release containing the following changes:
+  - Expose if_none_match optional parameters for upsert_digital_twin and upsert_relationship APIs
```

Comment:
We could probably rephrase this to something like:
- Added `etag` and `match_condition` parameters to `upsert_digital_twin` and `upsert_relationship` APIs to support conditional operation.

Reply:
Thanks - Added.